module/zfs/zfs_ioctl.c

```diff
@@ -5355,6 +5355,7 @@ zfs_ioc_tmp_snapshot(zfs_cmd_t *zc)
  * zc_name		name of "to" snapshot
  * zc_value		name of "from" snapshot
  * zc_cookie		file descriptor to write diff data on
+ * zc_flags		set if blockwise diff is requested
```

Comment:
Let's not commit all 32-bits to the blockwise diff flag and instead mask 0x1 for this purpose.